### DISCUSSION TITLE
Why dependencies are being installed like this ?

### DISCUSSION THREAD

**Discussion by jpkontreras** [vercel/next.js#57028] [Help]
### Summary

Im creating a new next app and with pnpm i notice some peer-deps issues and when i went to check package.json i found dependencies installed like this 

```
  "dependencies": {
    "react": "^18",
    "react-dom": "^18",
    "next": "13.5.6"
  },
  "devDependencies": {
    "typescript": "^5",
    "@types/node": "^20",
    "@types/react": "^18",
    "@types/react-dom": "^18",
    "autoprefixer": "^10",
    "postcss": "^8",
    "tailwindcss": "^3",
    "eslint": "^8",
    "eslint-config-next": "13.5.6"
  }
```
Why not use the exact version? what are you trying to do ?
Fyi this is breaking` pnpm create next-app` with the tailwind option enabled

**Comment by blentz100** [comment]
@jpkontreras Here is a good explanation of what's going on there: [Semver explained - why is there a caret (^) in my package.json?](https://bytearcher.com/articles/semver-explained-why-theres-a-caret-in-my-package-json/)

When you say this is breaking `pnpm create next-app`, can you provide more details as to how you know it's breaking?

**Comment by jpkontreras** [comment]
https://github.com/vercel/next.js/assets/7862089/aa74474f-6660-4bb9-bae5-634eddee489e

here look at it yourself... also please make your guys read the semver link you posted it might help them
